src/sssd.py

```diff
@@ -4,13 +4,14 @@
 
 """Provides sssd class to control sssd."""
 
-import base64
 import logging
 import os
+import pathlib
 import subprocess
 
 from charms.operator_libs_linux.v0 import apt
 from charms.operator_libs_linux.v1 import systemd
+from jinja2 import Template
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Since we removed the `__getattr__` code in glauth-operator, we should most likely remove it from here as well.

Reply:
True, should be consistent good catch. Just pushed an update.